crates/uv-distribution-filename/src/wheel.rs

```diff
@@ -202,25 +216,58 @@ impl WheelFilename {
                     .map_err(|err| WheelFilenameError::InvalidBuildTag(filename.to_string(), err))
             })
             .transpose()?;
+
+        let tags = if build_tag.is_some()
+            || python_tag.contains('.')
+            || abi_tag.contains('.')
+            || platform_tag.contains('.')
+        {
+            WheelTag::Large {
+                large: Box::new(WheelTagLarge {
+                    build_tag,
```

Comment:
Fwiw if there comes a case in the future where we want to pass around the compatibility tag separately from the wheel filename, we have to grow the `WheelFilename` a bit again by moving the build tag back in (but for now, we don't have that use case)